src/coreclr/jit/target.h

```diff
@@ -239,6 +239,9 @@ typedef unsigned __int64 regMaskSmall;
 typedef _regNumber_enum regNumber;
 typedef unsigned char   regNumberSmall;
 
+//TODO-LLVM: delete this when we are merged forward enough that it is no longer present
+#define SCALED_ADDR_MODES 1
+
```

Comment:
```suggestion
```
Deleted (in https://github.com/dotnet/runtime/pull/60808, which should be included in the commit stream here).

Reply:
Deleted now